spatialpandas/dask.py

```diff
@@ -200,8 +200,16 @@ def pack_partitions(self, npartitions=None, p=15, shuffle='tasks'):
         return ddf
 
     def pack_partitions_to_parquet(
-            self, path, filesystem=None, npartitions=None, p=15, compression="snappy",
-            tempdir_format=None, _retry_args=None
+        self,
+        path,
+        filesystem=None,
+        storage_options=None,
+        npartitions=None,
+        p=15,
+        compression="snappy",
+        tempdir_format=None,
+        _retry_args=None,
+        engine_kwargs=None,
```

Comment:
Presumably better to keep existing arguments in the same order, in case someone used positional args here (unlikely, in such a long list, of course!)?

Reply:
Good point, agreed.